docker/alias/alias_computation_test.py

```diff
@@ -0,0 +1,130 @@
+
+TEST_DATA_DIR = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), 'testdata')
+
+
+class AliasTest(unittest.TestCase, tests.ExpectationTest(TEST_DATA_DIR)):
+  """Alias tests."""
+
+  def test_basic(self):
+    """Tests basic case."""
+    osv.AliasGroup(
+        bug_ids=['aaa-123', 'aaa-124'],
+        last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    alias_computation.main()
+    bug_ids = osv.AliasGroup.query(
+        osv.AliasGroup.bug_ids == 'aaa-123').get().bug_ids
+    self.assertEqual(['aaa-123', 'aaa-124'], bug_ids)
+
+  def test_update_alias_group(self):
+    """Tests updating an existing alias group."""
+    osv.AliasGroup(
+        bug_ids=['bbb-123', 'bbb-234'],
+        last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    osv.Bug(
+        id='bbb-123',
+        db_id='bbb-123',
+        aliases=['bbb-345', 'bbb-456'],
+        status=1,
+        source='test',
+        public=True,
+        import_last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    osv.Bug(
+        id='bbb-789',
+        db_id='bbb-789',
+        aliases=['bbb-456'],
+        status=1,
+        source='test',
+        public=True,
+        import_last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    alias_computation.main()
+    alias_group = osv.AliasGroup.query(
+        osv.AliasGroup.bug_ids == 'bbb-123').get()
+    self.assertEqual(['bbb-123', 'bbb-234', 'bbb-345', 'bbb-456', 'bbb-789'],
+                     alias_group.bug_ids)
+    self.assertNotEqual(
+        datetime.datetime(2023, 1, 1), alias_group.last_modified)
+
+  def test_create_alias_group(self):
+    """Tests adding a new alias group."""
+    osv.Bug(
+        id='test-123',
+        db_id='test-123',
+        aliases=['test-124'],
+        status=1,
+        source='test',
+        public=True,
+        import_last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    osv.Bug(
+        id='test-222',
+        db_id='test-222',
+        aliases=['test-124'],
+        status=1,
+        source='test',
+        public=True,
+        import_last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    alias_computation.main()
+    alias_group = osv.AliasGroup.query(
+        osv.AliasGroup.bug_ids == 'test-123').get()
+    self.assertIsNotNone(alias_group)
+    self.assertEqual(['test-123', 'test-124', 'test-222'], alias_group.bug_ids)
+
+  def test_delete_alias_group(self):
+    """Tests deleting alias groups that only has one vulnerability."""
+    osv.Bug(
+        id='ccc-123',
+        db_id='ccc-123',
+        status=1,
+        source='test',
+        public=True,
+        import_last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    osv.AliasGroup(
+        bug_ids=['ccc-123'],
+        last_modified=datetime.datetime(2023, 1, 1),
+    ).put()
+    alias_computation.main()
+    alias_group = osv.AliasGroup.query(
+        osv.AliasGroup.bug_ids == 'ccc-123').get()
+    self.assertIsNone(alias_group)
+
+
```

Comment:
Can you also test for the case where we need to split an alias group into two?
e.g. have `A -> B -> C -> D` and remove the `B -> C` alias to get `A -> B` and `C -> D`